Makefile

```diff
@@ -1,6 +1,8 @@
+VERSION := $(shell git describe --tags --abbrev=0)
+
 build:
 	@go mod tidy
```

Comment:
I know it's not part of the PR, but just noticed this. since you are here - can we drop go mod tidy from `build`? we don't want the code to change right before we build